metals/src/main/scala/scala/meta/internal/metals/formatting/MultilineString.scala

```diff
@@ -227,11 +262,30 @@ case class MultilineString(userConfig: () => UserConfiguration)
       position: Position,
   ): Boolean = {
     val lineBefore = splitLines(position.getLine - 1)
-    getIndexOfLastQuote(lineBefore).exists { case (_, quoteClosed) =>
+    getIndexOfLastOpenQuote(lineBefore).exists { case (_, quoteClosed) =>
       !quoteClosed
     }
   }
 
+  private def wasTripleQuoted(
+      splitLines: Array[String],
+      position: Position,
+  ): Boolean = {
+    var closedFromPreviousLines = true
+    var existed = false
+    for (i <- 0 until position.getLine()) {
```

Comment:
It's a pity we need to go over every line, but I don't see a way around it :thinking: